docs/current/guides/snippets/work-with-host-filesystem/glob-pattern/index.mts

```diff
@@ -0,0 +1,35 @@
+import { connect, Client } from "@dagger.io/dagger"
+import * as fs from "fs"
+import * as os from "os"
+import * as path from "path"
+
+const dir = os.tmpdir()
```

Comment:
There is one issue, I found here in my my system is that it was failing with a wierd issue.
Logs: https://gist.github.com/JammUtkarsh/6424109b62f8c749aa2970c155039f34

When I changed this line to `const dir = path.resolve();`, it seems to be working file.

I request the maintainers to cross-verify on thier system as well.

My Configurations:
**OS**: darwin21.3.0
**NodeJS Version**: v20.5.1

Reply:
Some error, I could fix the issue by using `dir` instead of `.` as host.

We might have an issue with NodeSDK and workdir